content/en/user-guide/integrations/gitpod/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "GitPod"
+tags: ["gitpod", "development-environment", "integrated-development-environment"]
+weight: 13
+description: >
+  Use GitPod's fully automated, ephemeral workspaces to develop & test your cloud applications with LocalStack
+---
+
+<img src="gitpod_logo.png" width="600px" alt="GitPod logo">
+
+## Overview
+
+Gitpod is an open-source remote development platform for remote development. It provides an automated setup with cloud-based, remote developer environments connected with a developer’s [editing experience of choice](https://www.gitpod.io/docs/references/ides-and-editors). Gitpod allow users to codify their developer environment as code. With project's codified, you can spin up a new workspace, start coding and throw away the workspace when they are done!
+
+## LocalStack on GitPod
+
+LocalStack allows you to setup a development environment with a cloud sandbox that can be used to test and develop cloud applications. Using GitPod's environment you can run a LocalStack container inside the runtime that allows you to instantiate their application on a code editor of their choice. You can then conveniently deploy their cloud application assets into LocalStack's cloud sandbox, to then preview the results.
+
+To configure LocalStack on GitPod, you would need to setup a `.gitpod.yml` on the root of your repository. The file configures your workspace and the environment that you would like to use. You can find more information on the [GitPod documentation](https://www.gitpod.io/docs/config-gitpod-file/).
+
+```yaml
+tasks:
+  - name: start-localstack
+    env:
+      AWS_ACCESS_KEY_ID: "test"
+      AWS_SECRET_ACCESS_KEY: "test"
+      AWS_DEFAULT_REGION: "us-east-1"
+    init: |
+      python -m pip install localstack
+    command: |
+      . ~/.bash_profile
+      docker network create localstack
+      localstack start
+  - name: install-awslocal
+    env:
+      AWS_ACCESS_KEY_ID: "test"
+      AWS_SECRET_ACCESS_KEY: "test"
+      AWS_DEFAULT_REGION: "us-east-1"
+    init: |
+      cd /workspace
+      curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+      unzip awscliv2.zip
+      sudo ./aws/install
+      cd $THEIA_WORKSPACE_ROOT
```

Comment:
Curious about the `$THEIA_WORKSPACE_ROOT` env variable - wasn't able to find much information, is this documented somewhere on Gitpod side?

Reply:
The reference I found for this is here: https://github.com/gitpod-io/gitpod/issues/9804#issuecomment-1121839026

Unfortunately, its not documented but is used to `cd` to the workspace root.